packages/admin/admin/src/FinalForm.tsx

```diff
@@ -35,8 +36,7 @@ interface IProps<FormValues = AnyObject> extends FormProps<FormValues> {
 }
 
 export function FinalForm<FormValues = AnyObject>(props: IProps<FormValues>) {
-    const client = useApolloClient();
-    const dirtyHandler = React.useContext(DirtyHandlerApiContext);
+    const { client } = React.useContext(getApolloContext());
```

Comment:
unrelated change to have a soft runtime dependency on apollo (client can be undefined now)

(that simplified story)